src/gui/PluginBrowser.cpp

```diff
@@ -287,4 +291,24 @@ void PluginDescWidget::mousePressEvent( QMouseEvent * _me )
 }
 
 
+void PluginDescWidget::contextMenuEvent(QContextMenuEvent* e)
+{
+	QMenu contextMenu(this);
+	contextMenu.addAction(
+		tr("Send to new instrument track"),
+		[=]{ openInNewInstrumentTrack(m_pluginKey.desc->name); }
+	);
+	contextMenu.exec(e->globalPos());
+}
+
+
+void PluginDescWidget::openInNewInstrumentTrack(QString value)
```

Comment:
Any reason why ``value`` cant be a ``const`` ref?
```suggestion
void PluginDescWidget::openInNewInstrumentTrack(const QString& value)
```

Reply:
As I understand it, Qt objects are copy-on-write so this a shallow reference until writing and then creates a copy. If we're not changing the contents of value then using `QString` is fine.

Reply:
I prefer `const QString&` to be consistent with the practice with `std::string`, but I don't consider this as a blocker.